src/stan/io/deserializer.hpp

```diff
@@ -606,6 +605,32 @@ class deserializer {
         this->read<Ret>(size - 1), lp);
   }
 
+  /**
+   * Return the next zero sum matrix of the specified dimensions (requires N-1 *
+   * M-1 unconstrained scalars), incrementing the specified reference with the
```

Comment:
Can you just note that it doesn't require a Jacobian adjustment rather than saying it "increments the specified reference"?